packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -176,7 +176,7 @@ export const Workbench: React.FunctionComponent<{
   const callTab: TabbedPaneTabModel = {
     id: 'call',
     title: 'Call',
-    render: () => <CallTab action={activeAction} sdkLanguage={sdkLanguage} />
+    render: () => <CallTab action={activeAction} executionStartTime={model?.startTime ?? 0} executionStartWallTime={model?.wallTime ?? 0} sdkLanguage={sdkLanguage} />
```

Comment:
I don't think "execution" helps here, start time usually applies to execution, maybe name it timeOrigin?